backend/de.metas.cucumber/src/test/resources/de/metas/cucumber/features/bankStatement/camt.53.001.04-Import.feature

```diff
@@ -48,53 +42,81 @@ Feature: import bank statement in camt.53.001.04 import format
 
   @from:cucumber
   @Id:S0337_100
-  Scenario: Import bank statement, identify org-account by IBAN and link one statement-line to a payment for a purchase invoice that is matched via documentNo
+  Scenario: Import bank statement, identify org-account by IBAN and link one statement-line to a payment for a sales invoice that is matched via documentNo
+
+    Given set sys config boolean value true for sys config de.metas.payment.esr.Enabled
+    And load C_DataImport:
+      | C_DataImport_ID.Identifier | OPT.C_DataImport_ID |
+      | di_1_S0337_100             | 540009              |
+    And metasfresh contains C_Bank:
+      | C_Bank_ID.Identifier | Name       | RoutingNo | SwiftCode   | C_DataImport_ID.Identifier |
+      | b_1_S0337_100        | bank_S0337 | 2234567   | AAAAAAAA82A | di_1_S0337_100             |
+	# change the bankaccount of the AD_Org bpartner ("metasfresh") to be an ESR-Account
+    And load C_BP_BankAccount:
+      | C_BP_BankAccount_ID.Identifier | OPT.C_BP_BankAccount_ID |
+      | bpb_1_S0337                    | 2000257                 |
+
+    And update C_BP_BankAccount:
+      | C_BP_BankAccount_ID.Identifier | OPT.C_Currency.ISO_Code | OPT.IsEsrAccount | OPT.AccountNo | OPT.ESR_RenderedAccountNo | OPT.IBAN              |
+      | bpb_1_S0337                    | CHF                     | Y                | 1234567890    | 123456789                 | CH3908704016075473007 |
 
     And metasfresh contains C_Invoice:
       | Identifier      | C_BPartner_ID.Identifier | C_DocTypeTarget_ID.Name | DateInvoiced | C_ConversionType_ID.Name | IsSOTrx | C_Currency.ISO_Code | OPT.DocumentNo   |
-      | inv_1_S0337_100 | bpartner_1_S0337         | Eingangsrechnung        | 2022-05-11   | Spot                     | false   | EUR                 | 511004_S0337_100 |
+      | inv_1_S0337_100 | bpartner_1_S0337         | Ausgangsrechnung        | 2022-05-11   | Spot                     | true    | EUR                 | 511004_S0337_100 |
```

Comment:
the invoice is in eur but the bankstatement-line is in CHF..so i guess somethingneeds to be wrong with the amounts-matching